client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { trim, isEmpty } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Gravatar from 'components/gravatar';
+import FollowButton from 'reader/follow-button';
+import Gridicon from 'gridicons';
+
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( '/', '' );
+
+function SettingsMenu() {
+	return (
+		<span className="reader-subscription-list-item__settings-menu">
+			<Gridicon
+				icon="cog"
+				size={ 20 }
+			/>
+			Settings
+		</span>
+	);
+}
+
+function ReaderSubscriptionListItem( {
+	isFollowing,
+	siteUrl,
+	siteTitle,
+	siteAuthor,
+	siteExcerpt,
+	className = '',
+	onSiteClick = () => {},
+	followSource,
+	lastUpdated,
+} ) {
+	const authorName = trim( `${ siteAuthor.first_name || '' } ${ siteAuthor.last_name || '' }` );
```

Comment:
Is there a `siteAuthor.name` or `siteAuthor.fullName` we could use instead? Names are weird, and a first name + last name concatenation won't always do the trick.

https://www.w3.org/International/questions/qa-personal-names